.template/addons/crud/lib/template.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+def copy_template_files
+  copy_non_tt_files
+  copy_tt_files
+end
+
+def copy_non_tt_files
+  # The reason we are excluding .tt files is because
+  # this directory method will actually apply the template and remove the .tt extension in the destination
+  directory 'lib/templates', exclude_pattern: /.*\.tt$/
+end
+
+def copy_tt_files
+  src_path = File.join(__dir__, 'templates')
+  dst_path = 'lib/templates'
+
+  Dir.glob("#{src_path}/**/*.tt").each do |file_source|
+    relative_path = file_source.gsub("#{src_path}/", '')
+
+    copy_file(file_source, File.join(dst_path, relative_path))
+  end
+end
```

Comment:
Can you try this? :) I'm not sure if that would work, but that would be more elegant 😇 

```suggestion
def copy_template_files
  ignore_tt do
    directory 'lib/templates', renderTemplates: false
  end
end
```

See how to create the `ignore_tt` method in: https://stackoverflow.com/a/74875526/1914241

=> The block `ignore_tt` seems easier to understand :)
``
    ignore_tt do
      directory(...)
    end
```

Reply:
I actually took the second approach idea below this one in the same post from him haha 😅 . I will give the first one a try to see how it goes. Thank you 💪

Reply:
It **WORKED,** thank you very much 🙇 

Fixed in https://github.com/nimblehq/rails-templates/pull/505/commits/b17ce32c7adcceb722c8be2beb744162885fd711